Tests/ViewsTests/AvatarViewTests.swift

```diff
@@ -50,7 +50,7 @@ class AvatarViewTests: XCTestCase {
     func testWithImage() {
         let avatar = Avatar(image: nil)
         avatarView.set(avatar: avatar)
-        XCTAssertEqual(avatar.initials, "?")
+        XCTAssertEqual(avatar.initials, "")
```

Comment:
We changed the default value at https://github.com/quipper/MessageKit/commit/de95cd4e85f91566b9cbf0e7b9d6e9f27fb443f0#diff-8272c0fbb141fe6938bead67a2365ce451c7ecb42e0107acc6443ce23e7d2581